agr_java_core/src/main/java/org/alliancegenome/api/entity/DiseaseAnnotationDocument.java

```diff
@@ -28,7 +28,7 @@ public class DiseaseAnnotationDocument extends SearchableItemDocument {
 	private Reference singleReference;
 	private Set<Reference> references;
 
-	private String relationNegation;
+	private String negationInclRelation;
```

Comment:
@oblodgett @markquintontulloch Now updated, and I've immediately opened a PR in the UI repo as well to match these changes: alliance-genome/agr_ui#1213